README.md

```diff
@@ -139,7 +139,7 @@ Here's a table with summary of supported `react-native` versions when Fabric is
 ## Usage with [react-navigation](https://github.com/react-navigation/react-navigation)
 
 > [!CAUTION]
-> NativeStack has been moved from react-native-screens/native-stack to @react-navigation/native since version v6. With react-native-screens v4 native stack v5 (react-native-screens/native-stack) is deprecated and marked for removal in the upcoming minor release, react-native-screens v4 will support only @react-navigation/native-stack v7.
+> JS API of the native stack has been moved from react-native-screens/native-stack to @react-navigation/native-stack since version v6. Currently, native stack v5 (still held by react-native-screens) is deprecated and will be removed in the upcoming major release. From now on, only @react-navigation/native-stack v7 will be supported by react-native-screens.
```

Comment:
My suggestions:

```suggestion
> JS API of the native stack has been moved from `react-native-screens/native-stack` to `@react-navigation/native-stack` since version v6. Currently, native stack v5 (imported from `react-native-screens/native-stack`) is deprecated and will be removed in the upcoming **minor** release. `react-native-screens` v4 will support only `@react-navigation/native-stack` v7.
```